spec/unit/plugins/darwin/platform_spec.rb

```diff
@@ -41,6 +41,11 @@
     expect(@plugin[:platform_version]).to eq("10.15.6")
   end
 
+  it "sets platform_version_extra to ProductVersionExtra" do
+    @plugin.run
+    expect(@plugin[:platform_version_extra]).to eq("(a)").or be_nil
```

Comment:
this isn't a meaningful test - it'll be null given the test data... but it'd also be null if you were parsing wrong and the code was broken.

You can have this test be "returns null if no PVE is available", but then add another test that changes the test data (currently set on line 25) and ensure it's parsed properly.

Thanks!

Reply:
@jaymzh Thanks for the guidance here. I'm not overly familiar with writing tests, but did what I could here. I've updated `sw_vers` to match more recent output on macOS as well. Please let me know if there's anything else I should do to get the PR approved.